pycbc/types/timeseries.py

```diff
@@ -184,6 +184,12 @@ def time_slice(self, start, end, mode='floor'):
         start_idx = float(start - self.start_time) * self.sample_rate
         end_idx = float(end - self.start_time) * self.sample_rate
 
+        if _numpy.isclose(start_idx, round(start_idx)):
+            start_idx = round(start_idx)
+
+        if _numpy.isclose(end_idx, round(end_idx)):
+            end_idx = round(end_idx)
+
```

Comment:
Could you update the PR description with a sentence about this change? Does it fix something triggered by your tests?

Reply:
@titodalcanton See updated description.